src/components/Game.js

```diff
@@ -52,26 +70,46 @@ const Game = () => {
 
   return (
     <div className="game">
-      <div className="game-info">
-        <h1>Tic-Tac-Toe</h1>
-        <div className="game-player">
-          <form>
-            <label>
-              Player X
-              <input type="text" onChange={changePlayerX} value={playerX} />
-            </label>
-            <label>
-              Player O
-              <input type="text" onChange={changePlayerO} value={playerO} />
-            </label>
-          </form>
+      <div className="game__info">
+        <h1 className="game__title">
+          Tic-Tac-Toe
+          <button onClick={showRules} className="button">
+            ?
+          </button>
+        </h1>
+        <div className="game__content">
+          <div className="game__player">
+            <div className="game__player--x">
+              <label htmlFor="playerX">Player X Name</label>
+              <input
+                id="playerX"
+                type="text"
+                onChange={changePlayerX}
+                value={playerX}
+              />
+            </div>
+            <div className="game__player--o">
+              <label htmlFor="playerO">Player O Name</label>
+              <input
+                id="playerO"
+                type="text"
+                onChange={changePlayerO}
+                value={playerO}
+              />
+            </div>
+          </div>
+
+          <p>{statusText}</p>
+          <Board
+            squares={current.squares}
+            onClick={index => handleClick(index)}
+          />
         </div>
-        <div className="game-status">{statusText}</div>
-        <Board
-          squares={current.squares}
-          onClick={index => handleClick(index)}
-        />
+        <button className="game__reset" onClick={resetGame}>
+          Reset
+        </button>
       </div>
+      {isShowingGameRules && <Rules onClick={hideRules} />}
```

Comment:
Maybe be more descriptive on the component and prop name here?

`<RulesModal onClose={hideRules}/>`

Reply:
Can you please explain the onClose event handler? I tried making this change, and I was not able to exit the modal.

Reply:
`onClose` wouldn't be the event handler - it would just be the prop name.  You would still have an `onClick` event handler call the `onClose` prop